app/javascript/packs/application.js

```diff
@@ -3,4 +3,23 @@ require("turbolinks").start()
 require("@rails/activestorage").start()
 require("channels")
 require("bootstrap/dist/js/bootstrap")
+require("@fortawesome/fontawesome-free")
 
+
+$(function () {
+  // テキストボックスでキーアップされた時に実行
+  $('.search-text').keyup(function () {
+    // テキストボックスの内容を取得し変数に入れる
+    search_text = $(".search-text").val();
```

Comment:
こちらにも `toLowerCase()` が必要ですね。
```suggestion
    search_text = $(".search-text").val().toLowerCase();
```